Utils/main.py

```diff
@@ -0,0 +1,72 @@
+import argparse
```

Comment:
Completely optional and up to your preference: Consider using something like [fire](https://github.com/google/python-fire) to help structure commandline usage. Doing this might simplify how the commandline arguments are accessed by your audience.

Reply:
I will probably make this a python package and not use a CLI for the moment